src/aura/gs_homepageTemplate/gs_homepageTemplate.cmp-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<AuraDefinitionBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>49.0</apiVersion>
+    <description>Lightning Homepage template for being used in Getting Started page</description>
```

Comment:
Update description to say: Lightning Homepage Template used on the NPSP Getting Started page.